docs/3.0.0-beta.x/advanced/customize-admin.md

```diff
@@ -29,9 +29,101 @@ By default, the administration panel is exposed via [http://localhost:1337/admin
 
 The panel will be available through [http://localhost:1337/dashboard](http://localhost:1337/dashboard) with the configurations above.
 
+---
+
 ### Development mode
 
-**_Currently not available_**
+to enable the front-end development mode you need to start your application using the `--watch` flag.
+
+```bash
+cd my-app
+strapi develop --watch
+```
+
+With this option you can
+
+#### Customizing the `strapi-admin` package
+
+All files added in `my-app/admin/src/` will either be replaced or added
+
+**Example: Changing the available locales of your application**
+
+```bash
+# Create both the admin and admin/src folders
+cd my-app && mkdir admin && mkdir admin/src
```

Comment:
`mkdir -p admin/src`

Reply:
(For clarification mk -p means to make any required parent directories so you can just provide the deepest child directory and any non-existing parents will be created as needed @soupette )